Tests/OptimizelyTests-Batch-iOS/EventDispatcherTests_Batch.swift

```diff
@@ -648,7 +648,7 @@ extension EventDispatcherTests_Batch {
 
         dispatchMultipleEvents([(kUrlA, batchEventA)])
         
-        wait(for: [eventDispatcher.exp!], timeout: 3)
+        wait(for: [eventDispatcher.exp!], timeout: 10)
```

Comment:
Where do we get these timeout values from? Are they environment dependent?

Reply:
No rule for setting this up - just make it large enough to avoid false-negative but small enough to catch failure fast. It is supposed to be completed in less than a sec, but I see tests intermittently fail in travis, which has been slowed down recently. It should work with the increased timeout.